rest/rest-resources/src/main/java/org/apache/brooklyn/rest/util/ScannerInjectHelper.java

```diff
@@ -15,13 +15,18 @@
  */
 package org.apache.brooklyn.rest.util;
 
+import io.swagger.jaxrs.config.SwaggerContextService;
+import io.swagger.jaxrs.config.SwaggerScannerLocator;
 import org.apache.brooklyn.rest.apidoc.RestApiResourceScanner;
 import org.apache.cxf.jaxrs.JAXRSServerFactoryBean;
 
 import io.swagger.config.ScannerFactory;
 
 public class ScannerInjectHelper {
     public void setServer(JAXRSServerFactoryBean server) {
-        RestApiResourceScanner.install(server.getResourceClasses());
+        RestApiResourceScanner scanner = new RestApiResourceScanner(server.getResourceClasses());
+        ScannerFactory.setScanner(scanner);
+        // Above method broken in Swagger 1.6.2
```

Comment:
What's the consequence of that broken method? Can you expand this comment please?